spec/lib/farscape/plugins_spec.rb

```diff
@@ -82,7 +88,7 @@ def call(env)
       it 'removes middleware when the source plugin is disabled' do
         plugin = {name: :Peacekeeper, type: :sebacean, middleware: [TestMiddleware::NoGetNoProblem]}
         Farscape.register_plugin(plugin)
-        Farscape.disable( :sebacean )
+        Farscape.disable!( :sebacean )
         expect(Farscape.middleware_stack).to be_none
```

Comment:
Appears you removed the definition of `middleware_stack`. Wouldn't this fail? If not where is `middleware_stack` defined?